### PR TITLE
use kinder-run.sh

### PR DESCRIPTION
**Pull request by sanchezl**
use kinder-run.sh

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml [hunk 1]
```diff
@@ -98,7 +98,7 @@ presubmits:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230222-b5208facd4-master
         command:
         - runner.sh
-        - "./test/e2e/test-kinder.sh"
+        - "../kubeadm/kinder/ci/kinder-run.sh"
```
Comment (BenTheElder) [inline config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml:101]: This seems like the wrong path, given the path_aliases. Won't you need ../../k8s.io/kubeadm?

Comment (sanchezl) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml:101]: I was following the  examples in other jobs, such as [this one](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-external-ca.yaml), which use the same `path_alias`.

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml:101]: so the working directory is the first extra_refs for periodics, for presubmit and periodics it's the repo under test

in this case that means for this job the working dir will be the kube-storage-vesrsion-migrator repo clone which is at sigs.k8s.io/kube-storage-version-migrator

(both will be something like $GOPATH/src/$path_alias)

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml:101]: i'm trying to find the relevant documentation but it's been moved in bulk recently and i've lost track of where this is covered if anywhere. it's a little confusing.

Comment (BenTheElder) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml:101]: Aha! https://docs.prow.k8s.io/docs/components/pod-utilities/#what-the-test-container-can-expect

Comment (sanchezl) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml]: Updated. Thanks!

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**